contracts/babylon/src/msg/contract.rs

```diff
@@ -32,6 +32,8 @@ pub struct InstantiateMsg {
     pub notify_cosmos_zone: bool,
     /// If set, this will instantiate a BTC light client contract
     pub btc_light_client_code_id: Option<u64>,
+    /// JSON encoded `InitialHeader` in hex.
+    pub btc_light_client_initial_header: String,
```

Comment:
`btc_light_client_msg` (the instantiation message for BTC light client) already contains the initial header so probably we don't need this field

Reply:
It's necessary when `btc_light_client_msg` is `None`.

Reply:
>It's necessary when btc_light_client_msg is None

I was trying to keep the original logic that attempted to construct an `InstantiateMsg` when `btc_light_client_msg` was `None`, but now I realize this is unnecessary after discussing with Gai. If `btc_light_client_msg` is None, we may error out immediately rather than trying to handle it implicitly.

- Before: If `btc_light_client_msg` is None, attempt to construct an `InstantiateMsg`.
- After: If `btc_light_client_msg` is None, return an error.